web/react/components/updating_time_since_counter.jsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+var Utils = require('../utils/utils.jsx');
+
+var Tooltip = ReactBootstrap.Tooltip;
+var OverlayTrigger = ReactBootstrap.OverlayTrigger;
+
+export default class UpdatingTimeSinceCounter extends React.Component {
```

Comment:
I think just `TimeSinceTooltip` would be fine, I think the initial assumption anyone would make is that it is already updating and even if they didn't it's very apparent by looking at the contents of the component. The assumption should be that every componenet is "real-time" updating